src/jetstream/plugins/cfapppush/deploy.go

```diff
@@ -623,7 +624,13 @@ func cloneRepository(cloneDetails CloneDetails, clientWebSocket *websocket.Conn,
 		return "", err
 	}
 
-	vcsGit := GetVCS()
+	var vcsGit *vcsCmd
+
+	if len(cloneDetails.AccessToken) > 0 {
+		vcsGit = GetVCS(withAccessToken(cloneDetails.AccessToken))
+	} else {
+		vcsGit = GetVCS()
+	}
```

Comment:
nitpick: personally prefer using defaults over branching for less clutter

```suggestion
	vcsGit := GetVCS()

	if len(cloneDetails.AccessToken) > 0 {
		vcsGit = GetVCS(withAccessToken(cloneDetails.AccessToken))
	}
```

Reply:
Sure, the additional allocation here is well worth the readability change